packages/httpd.rb

```diff
@@ -11,6 +11,7 @@ class Httpd < Package
   depends_on 'aprutil'
```

Comment:
It is outside of this PR.  I don't want to optimize acyclic paths of dependencies to avoid causing new problem by this PR.